images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -133,6 +133,47 @@ _base_storage_overrides:
         hdfs:
           enabled: false
 
+  gcs:
+    storage:
+      hive:
+        # gcs storage defaults
+        gcs:
+          # Intentionally an invalid default name to ensure that this is overridden, and obvious when it's not.
+          container: "-unconfigured-metering-container-name-"
```

Comment:
I believe GCS uses the "bucket" terminology so we may wish to use `bucket` for the config option here.